pcdsdevices/sqr1.py

```diff
@@ -140,6 +141,17 @@ def move(
             self._sync_setpoints()
         return super().move(position, wait, timeout, moved_cb)
 
+    def set(self,
+            new_position: typing.Any,
+            *,
+            timeout: float = None,
+            moved_cb: typing.Callable = None) -> StatusBase:
```

Comment:
- Missing docstring to explain the override
- the `wait` parameter from positioner base has vanished, this could cause issues if upstream code calls this with explicit `wait=False`